### PR TITLE
Handling falling pieces

### PR DESCRIPTION
**Pull request by mineiwik**

### CODE REVIEW

File: scripts/board.gd [hunk 1]
```diff
@@ -30,6 +31,8 @@ func _ready():
 	self._draw_board()
 
 func get_piece_by_id(piece_id: String) -> Piece:
+	if not piece_id in self._pieces:
+		return null
 	return self._pieces[piece_id]
```
Comment (siku2) [inline scripts/board.gd]: get returns null if the key doesn't exist
```suggestion
	return self._pieces.get(piece_id)
```

File: scripts/board.gd [hunk 2]
```diff
@@ -149,6 +163,7 @@ func _draw_board():
 		if tile_type != "floor":
 			continue
```
Comment (siku2) [inline scripts/board.gd]: Just noticed that this behaviour is wrong. Tiles should still be created even if tile_type == "void", but they shouldn't be added to the floor_coordinates dict.

The reason the tile_type even exists is so we can assign textures to the void. Maybe something like a weird wormhole texture for a hole in the middle of the board.

Comment (siku2) [reply scripts/board.gd]: Doesn't need to be changed in this PR though.

File: scripts/board.gd (hunk 1 above)
Comment (mineiwik) [reply scripts/board.gd]: bda5a8d

File: scripts/board.gd (hunk 2 above)
Comment (mineiwik) [reply scripts/board.gd]: Added in bda5a8d - we now need to have a "void" texture...

**Review by siku2 — APPROVED** [review]

**Pull request merged**